samples/nrf9160/lte_ble_gateway/src/main.c

```diff
@@ -22,22 +22,27 @@
 #include "alarm.h"
 
 /* Interval in milliseconds between each time status LEDs are updated. */
-#define LEDS_UPDATE_INTERVAL	        500
+#define LEDS_UPDATE_INTERVAL 500
 
 /* Interval in microseconds between each time LEDs are updated when indicating
  * that an error has occurred.
  */
-#define LEDS_ERROR_UPDATE_INTERVAL      250000
+#define LEDS_ERROR_UPDATE_INTERVAL 250000
 
-#define BUTTON_1			BIT(0)
-#define BUTTON_2			BIT(1)
-#define SWITCH_1			BIT(2)
-#define SWITCH_2			BIT(3)
+#define BUTTON_1 BIT(0)
+#define BUTTON_2 BIT(1)
+#define SWITCH_1 BIT(2)
+#define SWITCH_2 BIT(3)
 
-#define LED_ON(x)			(x)
-#define LED_BLINK(x)			((x) << 8)
-#define LED_GET_ON(x)			((x) & 0xFF)
-#define LED_GET_BLINK(x)		(((x) >> 8) & 0xFF)
+#define LED_ON(x) (x)
+#define LED_BLINK(x) ((x) << 8)
+#define LED_GET_ON(x) ((x)&0xFF)
+#define LED_GET_BLINK(x) (((x) >> 8) & 0xFF)
```

Comment:
Personally find the macros more readble when they're aligned, but clang-format doesn't agree apparently :)

Reply:
i agree.